docs/release/release_0.0.1.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
1. can you remind me what the warning was that made this necessary? 
2. im confused why these don't show up in toc tree as you say, they are [here](https://graspy.neurodata.io/news.html) on the docs. are we currently not using toc tree somehow? reason i ask is the `:orphan:` thing seems a bit of a hack (just at a glance) and if we can do something to properly include them in a toc that would be better, but i could be wrong

Reply:
The warning was:
`11:27:56 AM: checking consistency... /opt/build/repo/docs/release/release_0.0.1.rst: WARNING: document isn't included in any toctree`
for each of the releases.  

So, they are on the docs, but only as direct links.  If you notice, you can't get to the release files from the table of contents, nor can you click 'next' or 'previous' when you are within one of the release docs (like you can with say, tutorials).  The :orphan: fix is a little hack-ish, but I was trying to preserve the current formatting.  I can try to include them in a toctree, but it may change the way things look.

Reply:
@bdpedigo Let me know which option you would prefer:
1. Keep current formatting, and use :orphan: to essentially suppress the warning
2. Include the release updates in a TOC tree, and potentially change the formatting

If (1), then I think this PR is ready for review.  Else if (2), I will try the fix and let you know.